Tests/Tests/CustomEventSpec.swift

```diff
@@ -0,0 +1,60 @@
+import Foundation
+import Quick
+import Nimble
+@testable import RInAppMessaging
+
+class CustomEventSpec: QuickSpec {
+    override func spec() {
+        describe("CustomEvent") {
+            let customAttributes =  [CustomAttribute(withKeyName: "Key1", withBoolValue: false),
+                                     CustomAttribute(withKeyName: "Key2", withDoubleValue: 3),
+                                     CustomAttribute(withKeyName: "Key3", withIntValue: 1)]
+            let customEvent = CustomEvent(withName: "Test", withCustomAttributes: customAttributes, timestamp: 60)
+
+            context("CustomEvent.analyticsParameters") {
+                it("will return dictionary value with analytics parameters") {
+                    expect(customEvent.analyticsParameters).toNot(beNil())
+                    expect(customEvent.analyticsParameters["eventName"]).to(beAKindOf(String.self))
+                    expect(customEvent.analyticsParameters["timestamp"]).to(beAKindOf(Int64.self))
```

Comment:
same here. Pease check for specific values and also add one more expectation for `["customAttributes"]` key